server/routes/__tests__/money.spec.js

```diff
@@ -13,10 +18,18 @@ const {
   consoleLogError,
 } = require('../../../test/test-helpers');
 
+const api = {
+  transactionHistory: /\/transaction-history/i,
+  pendingTransactions: /\/transaction-history\?account_code=cash&transaction_type=(HOA|WHF)/i,
+  agencies: /\/agencies\/type\/INST\?activeOnly=false/i,
+  balances: /\/balances/i,
+  damageObligations: /\/damage-obligations/i,
+};
```

Comment:
I'm not sure we gain anything by wrapping these in an object? I guess its purely for grouping _like_ constants and no other reason?

Reply:
Provides a namespace... the main reason was that vscode gives auto complete so you can type `api.` and it would pop up just the list of actual routes.

The same regexs are used by the client tests.